pyriemann_qiskit/utils/docplex.py

```diff
@@ -645,17 +660,85 @@ def prepare_model(qp):
                 # print(scaler.data_min_, scaler.data_max_)
                 scalers.append(scaler)
                 v.vartype = VarType.BINARY
-        return scalers
+                v.lowerbound = 0
+                v.upperbound = 1
+        conv = LinearEqualityToPenalty()
+        qp = conv.convert(qp)
+
+        return qp, scalers
+
+    """
+    Parameters
+    ----------
+    prob : Model
+        An instance of the docplex model [1]_
+    channels : list
+        The list of channels. A channel can be any Python object,
+        such as channels'name or number but None.
+    name : string
+        An custom name for the variable. The name is used internally by docplex
+        and may appear if your print the model to a file for example.
+
+    Returns
+    -------
+    docplex_covmat : dict
+        A square matrix of continuous decision variables representing
+        our covariance matrix.
+
+    See Also
+    -----
+    square_cont_mat_var
+
+    Notes
+    -----
+    .. versionadded:: 0.4.0
+
+    References
+    ----------
+    .. [1] \
+        http://ibmdecisionoptimization.github.io/docplex-doc/mp/_modules/docplex/mp/model.html#Model
+
+    """
```

Comment:
What class or function does this documentation correspond to?

Reply:
This is a link to the `docplex.mp.model` implementation. 
May be not too much useful after all.

Reply:
It is referenced here: 
![image](https://github.com/user-attachments/assets/d1f4f52c-5333-4824-adee-ab8c1cd8e86d)
Do you want me to remove it?

Reply:
No. My point is that I don't understand what class (or function) this docstring documents.

Normally, it is
```
def new_func(arg0, arg1):
    """ HERE, DOCUMENTATION """
    do_stuff
    return smart_result
```

Reply:
Oh, ok.